src/helpers/ERC721Proof.ts

```diff
@@ -106,10 +106,11 @@ export default class ERC721Proof
   async generateInputs(
     ownershipSignature: Signature,
     balanceSignature: BalanceSignature,
-    nullifierSignature: string,
     eddsaPublicKey: PublicKey
   ) {
-    const { r: r2, s: s2 } = utils.splitSignature(nullifierSignature)
+    const randomNumbersArray = crypto.getRandomValues(new BigUint64Array(2))
```

Comment:
How about doing it like this? ```const [r2, s2] = crypto.getRandomValues(new BigUint64Array(2))```

Reply:
This would be better, but returns r2 and s2 in the form "{xxxx}n" since it's a BigInt array - I'd have to cast to a string after wards to get rid of the n anyways, which would look equally as messy 

Ie, it'd look like something pretty similar to what we have now, unless there's a way to cast the BigInt values to strings in-place
```
var [r2, s2] = crypto.getRandomValues(new BigUint64Array(2))
var r2 = r2.toString()
var s2 = s2.toString()
```

Reply:
The nullifier takes bigInt, disregard what I said before. Thanks!